packages/graphback-cli/src/components/generate.ts

```diff
@@ -49,15 +50,37 @@ export const generateUsingPlugins = async (cliFlags: CliFlags) => {
   };
   const debouncedExec = debounce(runGeneration, 100);
 
+  const modelPath = getModelPath(config.dirpath, graphbackConfig.model)
+
   if (cliFlags.watch) {
-    chokidar.watch(graphbackConfig.model, {
+    chokidar.watch(modelPath, {
       persistent: true,
       cwd: config.dirpath,
     }).on('all', debouncedExec);
   } else {
-    const schemaDocument = await project.loadSchema(join(config.dirpath, graphbackConfig.model, '/**/*.graphql'));
+    
+    const schemaDocument = await project.loadSchema(modelPath);
     const generator = new GraphbackGenerator(schemaDocument, graphbackConfig);
     generator.generateSourceCode();
   }
 };
 
+/**
+ * 
+ * Formats the model path or list of paths to ensure
+ *   * only paths from inside the project are allowed
+ *   * if a directory is supplied *.graphql is appended
+ * 
+ * @param baseDir the base directory that should be appended to the path or paths
+ * @param model a path or a list of paths
+ */
+function getModelPath(baseDir: string, model: string | Array<string>): string | Array<string> {
+  if (typeof model === 'string' && existsSync(model) && lstatSync(model).isDirectory()) {
+    return join(baseDir, model, '/**/*.graphql')
```

Comment:
Thanks to graphql-config schema loaders, we should be able to use multiple input forms. This would mean that using `./model` will always assume `.graphql` file. This is absolutely fine - I think our documentation and templates should focus on specifying glob format so that this clear.